Sources/Msg.php

```diff
@@ -1231,8 +1231,7 @@ public static function spellCheck(): void
 
 		// Free resources for enchant...
 		if (isset(Utils::$context['enchant_broker'])) {
-			enchant_broker_free_dict($dict);
-			enchant_broker_free(Utils::$context['enchant_broker']);
+			unset($dict, Utils::$context['enchant_broker']);
```

Comment:
Note here, the enchant_*free functions are deprecated, they recommend just destroying the object.